<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit_Journal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_CurrentSemester">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt; color:#9b98b1;&quot;&gt;Spring, 2023&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LoginRegister">
    <property name="geometry">
     <rect>
      <x>679</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Login/Register</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>151</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_Courses_2">
     <item>
      <widget class="QLabel" name="label_Courses">
       <property name="text">
        <string>COURSES</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_Courses">
       <item>
        <widget class="QLineEdit" name="lineEdit_Coures1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Coures2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Coures3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Coures4"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>61</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_Grades">
     <item>
      <widget class="QLabel" name="label_Grades">
       <property name="text">
        <string>GRADE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_Credits">
       <item>
        <widget class="QLineEdit" name="lineEdit_Grade1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Grade2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Grade3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Grade4"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>160</y>
      <width>51</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_LetterGrades">
     <item>
      <widget class="QLabel" name="label_LetterGrade1">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_LetterGrade2">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_LetterGrade3">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_LetterGrade4">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>290</y>
      <width>178</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_SemesterGPAinfo">
     <item>
      <widget class="QLabel" name="label_SemesterGPAPrompt">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; color:#ef9838;&quot;&gt;SEMESTER GPA&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_SemesterGPA_Display">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; color:#ef9838;&quot;&gt;3.70&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionRedo"/>
    <addaction name="actionUndo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="separator"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="separator"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Resource2.qrc">
     <normaloff>:/rec2/stationery-icons-2022-11-22-22-38-16-utc/png/046-notebook.png</normaloff>:/rec2/stationery-icons-2022-11-22-22-38-16-utc/png/046-notebook.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resource2.qrc">
     <normaloff>:/rec2/stationery-icons-2022-11-22-22-38-16-utc/png/021-folder.png</normaloff>:/rec2/stationery-icons-2022-11-22-22-38-16-utc/png/021-folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resource2.qrc">
     <normaloff>:/rec2/stationery-icons-2022-11-22-22-38-16-utc/png/012-envelope.png</normaloff>:/rec2/stationery-icons-2022-11-22-22-38-16-utc/png/012-envelope.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/Back .png</normaloff>:/rec/Back .png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Resource2.qrc">
     <normaloff>:/rec2/stationery-icons-2022-11-22-22-38-16-utc/png/002-scissors.png</normaloff>:/rec2/stationery-icons-2022-11-22-22-38-16-utc/png/002-scissors.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Resource2.qrc">
     <normaloff>:/rec2/stationery-icons-2022-11-22-22-38-16-utc/png/023-clipboard.png</normaloff>:/rec2/stationery-icons-2022-11-22-22-38-16-utc/png/023-clipboard.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Resource2.qrc">
     <normaloff>:/rec2/stationery-icons-2022-11-22-22-38-16-utc/png/016-pinned notes.png</normaloff>:/rec2/stationery-icons-2022-11-22-22-38-16-utc/png/016-pinned notes.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/Bluetooth.png</normaloff>:/rec/Bluetooth.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/Alarm.png</normaloff>:/rec/Alarm.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="Resource2.qrc">
     <normaloff>:/rec2/stationery-icons-2022-11-22-22-38-16-utc/png/049-stationery.png</normaloff>:/rec2/stationery-icons-2022-11-22-22-38-16-utc/png/049-stationery.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resource2.qrc"/>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
